lib/java/persistance/src/main/java/org/enso/persist/PerUtils.java

```diff
@@ -1,8 +1,12 @@
 package org.enso.persist;
 
+import java.util.logging.Logger;
+
 final class PerUtils {
   private PerUtils() {}
 
+  static final Logger LOG = Logger.getLogger(Persistance.class.getPackageName());
```

Comment:
Hubert, there is #8644 - you can resolve it by providing a guidance during this review.

Reply:
@hubertp, I haven't heard any guidance on using logger properly to avoid pitfalls described by 
- #8644

Anyway, this PR doesn't make the current state worse. Integrating and hoping you'll be able to improve the logging here when processing #8644